bread/layout/components/datatable.py

```diff
@@ -566,10 +569,55 @@ def searchbar(search_urlparameter: str):
                 ),
             ),
             method="GET",
+            name="browseviewsearchform",
         ),
+        hg.LINK(
+            rel="stylesheet",
+            type="text/css",
+            href=staticfiles_storage.url("djangoql/css/completion.css"),
+        ),
+        hg.SCRIPT(src=staticfiles_storage.url("djangoql/js/completion.js")),
```

Comment:
If these asset files are only used within the BrowseViewSearch then we should move them over there.

Reply:
OK.